test/spec/proxySpec.js

```diff
@@ -1148,6 +1146,86 @@ describe('proxy', function() {
       // }, 100);
     });
   });
+  describe('Already proxified values', function() {
+    it("shouldn't re-proxify (it should deep clone) proxified values from different instances", function() {
```

Comment:
> But IMO, I'd rather have:
> ```js
> ProxyB.proxifiedObject.add // undefined
> ```
> Than have two proxies emitting patches together.

TBH, I'd prefer two proxies emitting pactches. If I created two proxies that eventually consiste same (sub)tree I would expect both to emit patches when something changes. What is the downside of that?
Especially given, that:
> our developers will never have two instances of JSONPatcherProxy.
-----
> You'll be able to sneak non-proxified objects into the proxified tree causing a lot of nightmarish bugs.

Could you give an example of such?
-----
 > On what occasions "our developers" can run into this?

I'd imagine the case of Vanilla/otherframework view like:

```html
<template>
  <script my-framework-integration>
    var script = document.currentScript;
    var palindromModel = script.model;
    script.addEventListener('model-changed',function(){/*...*/});
    // ...
    // UI reaction to clicking on "Start review" button
    var reviewState = myApp.startReview(); // {status: 'pending', reviewer: 'tomalec', linesChecked: 0} 
    palindromModel.reviewObj = reviewState;
    reviewState.linesChecked = myApp.currentLine;
  </script>
</template>